packages/lib/src/components/Select/Select.tsx

```diff
@@ -174,6 +178,7 @@ const SelectComponent: ForwardRefRenderFunction<
   useLayoutEffect(() => {
     const ElementsWithScrolls = (function () {
       const getComputedStyle =
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
@pruthvi2103 @Raalzz I need help in figuring out the types in this function.

Reply:
Lets move this to tech debt for now and release this version